scripts/cdn_deploy.js

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env node
+
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const path = require('path');
+const {exec} = require('child_process');
+const argv = require('minimist')(process.argv.slice(2));
+const S3_DEFAULT_BUCKET = "cdn.ably.io";
+const S3_DEFAULT_ROOT = "lib";
+
+async function run() {
+
+	let config = {
```

Comment:
Is this tab indented? Would prefer 2 space if possible.

Reply:
The whole repo is generall tab indented (see `.editorconfig`) I would love it if we could use 2 space indents but if we're enforcing a rule I'd like for it to be enforced across the entire codebase and it's just not worth overwriting all the git blame history for something like this 🙁

Reply:
Fair point, though this is new code and if we know we will be adopting 2 space in near future then this is a file that will then not need to be touched at that point.

Reply:
If we are going to change the indentation, I think it would make more sense to do it all in one go whilst changing the .editorconfig and any linting rules at the same time to keep it all consistent